src/preferencesControl.ts

```diff
@@ -37,8 +37,8 @@ export class PreferencesControl {
     public createPreferencesDialog(): void {
         let cookieModalInnerHtml = `
         <div role="presentation" tabindex="-1"></div>
-        <div role="dialog" aria-modal="true" aria-label="${ HtmlTools.escapeHtml(this.textResources.preferencesDialogTitle) }" class="${ styles.modalContainer }" tabindex="-1">
-            <button aria-label="${ HtmlTools.escapeHtml(this.textResources.preferencesDialogCloseLabel) }" class="${ styles.closeModalIcon }" tabindex="0">&#x2715;</button>
+        <div role="dialog" aria-modal="true" aria-label="${ HtmlTools.escapeHtml(this.textResources.preferencesDialogTitle) }" class="${ styles.modalContainer }" tabindex="0">
```

Comment:
This approach creates one problem: you have a focusable dialog. I.e. I can stop with TABs on the dialog itself, which is totally useless. An element should be focusable only if it's actionable: you can type text in it, you can press a button, can change its state somehow. In the case of a dialog you can't do anything, and it shouldn't be focusable because of that.